saleor/userprofile/i18n.py

```diff
@@ -5,14 +5,27 @@
 import i18naddress
 from django import forms
 from django.forms.forms import BoundField
-from django.utils.translation import pgettext, pgettext_lazy
+from django.forms import Select, TextInput
 from django_countries.data import COUNTRIES
+from django.utils.translation import pgettext_lazy
+from phonenumber_field.widgets import (
+    PhoneNumberPrefixWidget, PhonePrefixSelect)
+from phonenumber_field.formfields import PhoneNumberField
+from phonenumbers import COUNTRY_CODE_TO_REGION_CODE
 
 from .models import Address
+from .validators import validate_possible_number
+
+
+PhoneNumberField.default_validators = [validate_possible_number]
+PhoneNumberField.to_python = lambda self, value: value
```

Comment:
Instead of this kind of patching, we should rather extend the original class and override the methods or fields that we want to customize. This would be also a good place to include relevant docstrings, explaining how the customization is different from the original implementation.